src/main/rst/03_webdriver.rst

```diff
@@ -356,6 +374,13 @@ Example of how to find an element that looks like this:
     @cheeses = $driver->find_elements('cheese', 'class');
 
 
+.. code-block:: javascript
+	
+	findElements(By.className("cheese")).then(function(cheeses){
```

Comment:
I had a dilemma on using that, glad you suggested the es6 one. Shall make the changes.